modules/uninstalling-olm-using-ansible.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * install_config/uninstall-olm-using-ansible.adoc
+
+[id="uninstalling-olm-using-ansible_{context}"]
+= Uninstalling Operator Lifecycle Manager using Ansible
+
+After installing your cluster, you can uninstall the Technology Preview Operator Framework by using he {product-title} `openshift-ansible` installer.
+
+.Prerequisites
+
+- An existing {product-title} 3.11 cluster
+- Access to the cluster using an account with `cluster-admin` permissions
+- Ansible playbooks provided by the latest `openshift-ansible` installer
+
+.Procedure
+
+. Add the following variables to your `config.yml` playbook:
++
+[source,yaml]
+----
+operator_lifecycle_manager_install=false
+operator_lifecycle_manager_remove=true
+----
+
+. Change to the playbook directory and run the OLM installation playbook to uninstall OLM using your inventory file:
++
+[source,terminal]
+----
```

Comment:
You can have only one command in a codeblock. Please split the step up, either into two separate steps or substeps.

Reply:
Addressed